js/google_ima.js

```diff
@@ -147,6 +147,7 @@ require("../html5-common/js/utils/utils.js");
         this.adPlaybackStarted = false;
         this.vcPlayRequested = false;
         this.savedVolume = -1;
+        this.useGoogleAdUI = false;
```

Comment:
this is purely a flag to turn on Google's UI for ads.  Our is turned off permanently for Google IMA ads.  To not block ui elements like the skip button.  The skip button is always there, but this flag enabled stuff like ad countdown and their 'learn more' button.